app/pages/analyst/application-review.tsx

```diff
@@ -59,14 +59,14 @@ class ApplicationReview extends Component<Props, State> {
           ) {
             id
             versionNumber
-            overrideJustification
             isCurrentVersion
             applicationRevisionStatus {
               id
               applicationRevisionStatus
             }
             ...ApplicationDetailsContainer_applicationRevision
             ...IncentiveCalculatorContainer_applicationRevision
+            ...ApplicationReviewValidationContainer_applicationRevision
```

Comment:
This is hard to notice (because it's just a TS error, the tests still pass), but the test query needs to be updated in `application-review.test.tsx` anytime we change the query on the page.